ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -238,7 +246,8 @@ const mapStateToProps = (state: GlobalState): ReduxMappedStateProps => ({
 
 const mapDispatchToProps = (dispatch: Dispatch): ReduxMappedDispatchProps => ({
   resetPin: () => dispatch(startPinReset()),
-  logout: () => dispatch(logoutRequest())
+  logout: () => dispatch(logoutRequest()),
+  clearCache: () => dispatch(clearCache())
```

Comment:
since it looks like there's no navigation involved, can we at least add a toast to give the user a feedback on what's happened?